docs/source/posts/2021/2021-06-14-week-2-sajag.rst

```diff
@@ -0,0 +1,42 @@
+First week of coding!
+=====================
+
+.. post:: June 14 2021
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Welcome to the second weekly check-in. I'll be sharing my progress for the first week of coding.
+
+What did you do this week?
+--------------------------
+I implemented the space filling model for proteins and created a PR for the same. Preview -
+
+.. image:: https://user-images.githubusercontent.com/65067354/121518963-b92cb580-ca0e-11eb-8232-3512edc04670.png
+   
+(protein rendered: `3pgk <https://www.rcsb.org/structure/3pgk>`_)
```

Comment:
`..image` has a caption attribute. Please, use this instead of the current way

Reply:
done (`.. figure` has caption image doesn't)